R/FilterWrapper.R

```diff
@@ -72,9 +76,24 @@
 #'   getFilteredFeatures(model)
 #' })
 #' print(r$extract)
+#'
+#' #usage of a custom thresholding function
+#'  biggest_gap = function(values, diff) {
+#'    # Calculate the largest difference between values that are 'diff' positions apart
```

Comment:
Could you please add the code for this? It's ok if it takes a few lines.

Reply:
OK